.azure-pipelines/templates/platform_pr_pipeline.yml

```diff
@@ -24,4 +25,22 @@ stages:
           script_parameter: |
             python -m mlops.${{ parameters.model_type }}.src.mlops_pipeline \
               --build_environment ${{parameters.exec_environment}} \
-              --wait_for_completion True
+              --wait_for_completion True \
+              --output_file run_id.txt
+      - task: AzureCLI@2
+        displayName: "Cancel AML Pipeline"
+        condition: canceled()
+        inputs:
+          azureSubscription: $(AZURE_RM_SVC_CONNECTION)
+          scriptType: bash
+          scriptLocation: inlineScript
+          inlineScript: |
+            echo "Pipeline was canceled. Attempting to cancel AML Pipeline..."
```

Comment:
Can you have this print out the `run_id`/`run_name` of the job it's trying to cancel?

Reply:
![image](https://github.com/user-attachments/assets/b42c1bcd-3a21-4bd5-b50b-6dd20c924342)